rse-teams.Rmd

```diff
@@ -104,25 +104,24 @@ While the former describes how contributors interact with each other,
 the latter dictates how the project can be used and redistributed.
 Moreover, if the law or a publication agreement prevents people from reading our work or using our software,
 they can't contribute, so license selection plays a role in the culture of your project.
-We may occasionally need to keep things closed in order to respect personal or commercial confidentiality,
-but we default to open to encourage participation and to match the values of the project participants.
+We may occasionally need to keep things closed in order to respect personally identifiable data or commercial confidentiality,
+but we default to open to encourage participation within open source projects and to match the values of the project participants.  Teams using GitHub and other similar tools should seriously consider what their default should be.  Those working strictly in software may not have any problem having everything open.  However, teams working with sensitive data, particularly teams with members new to git systems, may wish to keep projects closed during active delelopment.  This allows the team to perform checks and make informed selections for what should be released publicly.
```

Comment:
One problem is that this chapter is really speaking directly to open source software projects, but also trying to be useful to large research labs.  Which isn't a bad thing, but sometimes the recommendations need to be different. 
I think open software projects should have a default to open.  Many research labs may also be completely fine having a default open.  However, there are some labs that deal with enough sensitive data in combination with newcomers to the git process that it could be exceedingly dangerous to have a default open. They may not even be able to use a private git repo to hold their data (I know my IRB data couldn't be in there).  So some labs may make the choice that their default is private, and that's a perfectly fine choice if the driving factors are the protection sensitive information within the data/code.
This also comes down to a large messaging problem I have with open science.  Openness is good, but there seems to be a binary set of judgements that anything not open is bad. I think we (like society we) need to be more inclusive in our messaging that not all data can or should be open.  But this is a different reason than for a software tool being collaboratively developed. So not a dig on your chapter or your perspective. We are so used to riding between these lines that detaciting our understanding of exceptions is important.

Reply:
I completely understand and agree with you on all counts. I'm wondering now about how to frame this: perhaps we can include a more general caveat at the top of this section differentiating considerations for open source projects vs. lab-based projects. There are a few other places I think we can make this distinction, too: in lab-based research, we don't really choose who we work with. 

At some point, we may need to include this in the broader narrative, but off the top of my head, this section is where that information is most essential.

Reply:
I think it's worth having a dedicated section about this somewhere. People working in this space need to navigate the information resources, which are often not coded properly for the audiences. I've seen people get turned off of open science stuff before because the reccs were for open source not science.  I can try and write up a suggested section for review, if we can firm up where it should go.